pkg/containerd/cmd/service/skanky-vendor.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# We only need the containerd client and its transitive dependencies
+# and we conveniently have a checkout already. We actually prefer to
+# reuse containerd's vendoring for consistency anyway.
+
+set -eu
+ctrd=$1
+mkdir -p vendor/github.com/containerd/containerd
+ln -nfs $ctrd/* vendor/github.com/containerd/containerd
+rm -f vendor/github.com/containerd/containerd/vendor
+
+for p in github.com/Microsoft/go-winio \
```

Comment:
We need containerd itself too, so it's not a simple `cp` of `vendor` but it might be simpler than this scheme.